src/hammer-vlsi/technology/asap7/__init__.py

```diff
@@ -191,4 +193,29 @@ def scale_gds_script(self, gds_file: str) -> str:
 gds_lib.write_gds('{gds_file}')
         """.format(cell_list_file=os.path.join(self.extracted_tarballs_dir, 'ASAP7_PDKandLIB.tar/ASAP7_PDKandLIB_v1p5/asap7libs_24.tar.bz2/asap7libs_24/gds/cell_list.txt'), gds_file=gds_file)
 
+    def get_tech_par_hooks(self, tool_name: str) -> List[HammerToolHookAction]:
+        hooks = {"innovus": [
+            HammerTool.make_post_insertion_hook("write_design", scale_final_gds)
+            ]}
+        return hooks.get(tool_name, [])
+
+def scale_final_gds(ht: HammerTool) -> bool:
+    assert isinstance(ht, HammerPlaceAndRouteTool)
+    assert isinstance(ht, CadenceTool)
```

Comment:
```suggestion
    assert isinstance(ht, CadenceTool), "scale_final_gds requires innovus"
```

^ actually should it be something more specific?